src/main/resources/openapi.json

```diff
@@ -1,286 +1,935 @@
+            },
+            "Offsets": {
+                "title": "Offsets",
+                "type": "object",
+                "properties": {
+                    "offsets": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/Offset"
+                        }
+                    }
+                },
+                "example": "{\n    \"offsets\": [\n        {\n            \"topic\": \"topic\",\n            \"partition\": 0,\n            \"offset\": 15\n        },\n        {\n            \"topic\": \"topic\",\n            \"partition\": 1,\n            \"offset\": 42\n        }\n    ]\n}"
+            },
+            "createdConsumer": {
+                "title": "Root Type for createdConsumer",
+                "description": "The root of the createdConsumer type's schema.",
+                "type": "object",
+                "properties": {
+                    "instance_id": {
+                        "description": "Unique ID for the consumer instance in the group",
+                        "type": "string"
+                    },
+                    "base_uri": {
+                        "description": "Base URI used to construct URIs for subsequent requests against this consumer instance",
+                        "type": "string"
+                    }
+                },
+                "example": "{\n    \"instance_id\": \"consumer1\",\n    \"base_uri\": \"http://localhost:8080/consumers/my-group/instances/consumer1\"\n}"
+            },
+            "OffsetsWithoutTopic": {
+                "title": "Root Type for OffsetsWithoutTopic",
+                "description": "The root of the OffsetsWithoutTopic type's schema.",
+                "type": "object",
+                "properties": {
+                    "offsets": {
+                        "type": "array",
+                        "items": {}
+                    }
+                },
+                "example": "{\n    \"offsets\": [\n        {\n            \"partition\": 2,\n            \"offset\": 0\n        },\n        {\n            \"partition\": 1,\n            \"offset\": 1\n        },\n        {\n            \"partition\": 2,\n            \"offset\": 2\n        }\n    ]\n}"
+            },
+            "Records2": {
```

Comment:
Why do we need two?

Reply:
`Records` contains nested `records` array. The `Records` contains a list of records. I know it is not ideal but I wanted to be accurate.

Reply:
I don't really think calling one of these things `Records2` ireally improves accuracy. Would it make sense to rename one of them `RecordList` or something like that, if one of them is really just a holder for an array of records?

Reply:
Why? `Records` defines an array type containing elements of type `Record`. Why we need `Records2`?